meta-balena-common/recipes-bsp/u-boot/files/balenaos_uboot_prod.cfg

```diff
@@ -1 +1,5 @@
 CONFIG_BOOTDELAY=-2
+CONFIG_DISABLE_CONSOLE=y
+CONFIG_SILENT_CONSOLE=y
+CONFIG_SILENT_CONSOLE_UPDATE_ON_RELOC=y
+CONFIG_SYS_DEVICE_NULLDEV=y
```

Comment:
Patch applied ok from the looks of it. I think that with your patch the above are not needed anymore? Or are they?

Reply:
Yes, patch appears to apply fine and in this case II'll update it to not take into consideration any of the above flags and remove them as well.

Reply:
Re-using for the moment this PR for testing build for all boards. Will open another one against master branch as soon as we have a variant that applies to all.